optimum/modeling_base.py

```diff
@@ -85,7 +85,6 @@ class PreTrainedModel(ABC):  # noqa: F811
 
 class OptimizedModel(PreTrainedModel):
     config_class = AutoConfig
-    load_tf_weights = None
```

Comment:
Could this result in an issue for previous transformers version like v4.29.0 ?

Reply:
we already updated min transformers version to 4.36

Reply:
I see v4.29.0 in https://github.com/huggingface/optimum/blob/v1.21.4/setup.py#L18, in any case asking to make sure that also works for the min supported transformers version

Reply:
oh I must have been looking at optimum-intel 😅. Let me run a test on older versions and check.

Reply:
tested and it doesn't seem to impact anything.

Reply:
thanks @IlyasMoutawwakil !